22_pipeslab/pipeslab.txt

```diff
@@ -2,19 +2,22 @@ For each of the following, type a single command line that performs:
 
   1. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner.
   Answer: 
-
+	ls –l |grep ‘^…x’ |wc -l
   2. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner, group and others.
   Answer: 
-
+	ls –l |grep ‘^…x..x..x’ |wc -l
   3. Print the number of lines from "ls -l" which represent paths that were modified today (no matter which day we run the command)
   Answer: 
-
+	ls -l |grep "$(date +%b)\s* $(date +%e)" | wc -l
   4. Modify (1) and (2) to count files only (no directories)
   Answer:
-
+	ls –l |grep ‘^-..x’ |wc –l
+	ls –l |grep ‘^-..x..x..x’ |wc -l
   5. The command /sbin/ifconfig shows information regarding the machine's network interfaces. Use the command to print only the IP address of the machine. Remember not to show the special loopback address 127.0.0.1
-
+	Ifconfig | grep inet |gerp –v 127.0.0.1
```

Comment:
I think that you need to ```cut``` and add a space to "inet" to make it work for ipv4.. but ```ifconfig``` is one of those commands that vary so much between systems.. so well done! :)

also a bit of nerd trivia - all addresses that start with 127. are loopback addresses ;)